vars/newmanExecute.groovy

```diff
@@ -114,6 +114,10 @@ void call(Map parameters = [:]) {
             dockerWorkspace: config.dockerWorkspace,
             stashContent: config.stashContent
         ) {
+            sh """
```

Comment:
You can use `sh returnStatus: true, script: '...'` to ignore exit codes

Reply:
Would you prefer that over piping stderr to `/dev/null`?

Reply:
✅ Done